.config/nvim/lua/plugins/extras/lang/tiltfile.lua

```diff
@@ -0,0 +1,35 @@
+-- Dependencies to install:
+-- tilt: brew install tilt (includes tilt_ls via `tilt lsp start`)
+-- buildifier: go install github.com/bazelbuild/buildtools/buildifier@latest (for starlark formatting/linting)
```

Comment:
Nah, don't want to get in the business of installing go + things. Remove this if this is the only way to install